src/ui/structure/container.tsx

```diff
@@ -22,10 +28,18 @@ interface Props {
   testID?: string
   children: ReactNode
   style?: StyleProp<ViewStyle>
+  safeArea?: boolean
 }
 
-export const Container: React.FC<Props> = props => (
-  <SafeAreaView testID={props.testID} style={[styles.container, props.style]}>
-    {props.children}
-  </SafeAreaView>
-)
+export const Container: React.FC<Props> = props => {
+  const ContainerView = props.safeArea === false ? View : SafeAreaView
```

Comment:
Why not just `props.safeArea ? SafeAreaView : View`?

Reply:
Because `SafeAreaView` should be rendered by default (even when `props.safeArea` is `undefined`). Thus, the `View` will only be rendered if the `safeArea` prop is passed as `false`. This is done to accommodate for screens such as the `InteractionScreen`, where the UI has to be full screen.

Reply:
The logic seems inverted. You can name the prop something like
`withoutSafeArea`, then check `props.withoutSafeArea ? View : SafeAreaView`. In this case, the component would still work the same.
So if you need want a safe area you use `<Wrapper>`, if no safe area you use `<Wrapper withoutSafeArea>` (which is a shorthand notation for `<Wrapper withoutSafeArea={true}>)`